core/src/test/scala/org/apache/spark/sql/delta/DeltaDataFrameHadoopOptionsSuite.scala

```diff
@@ -29,6 +29,8 @@ class DeltaDataFrameHadoopOptionsSuite extends QueryTest with SQLTestUtils with
 
   protected override def sparkConf =
     super.sparkConf.set("spark.delta.logStore.fake.impl", classOf[LocalLogStore].getName)
+      // disable the spark conf check
```

Comment:
Instead of using this config, could you extend `DeltaSQLCommandTest` instead when it's working? Same for other tests. This config is added mostly for safety.

Reply:
I refactored this test class `DeltaDataFrameHadoopOptionsSuite.scala` to extend `DeltaSQLCommandTest` and all tests ran successfully. However, the question I have is that do we need all Delta test class to extend `DeltaSQLCommandTest` even if they actually don't need the SQL Parser and the catalog config (e.g. `DeltaLog`). When I refactored the test classes, my approach was - if the test class doesn't already extend `DeltaSQLCommandTest`, then set the boolean flag to skip the spark conf check, therefore not requiring the `DeltaSQLCommandTest` trait. Let me know if this makes sense.